api/lib/domain/errors.js

```diff
@@ -1105,6 +1105,26 @@ class UnexpectedOidcStateError extends DomainError {
   }
 }
 
+class OidcMissingFieldsError extends DomainError {
+  constructor(message = 'Mandatory information about the user is missing.', code, meta) {
```

Comment:
Il serait préférable que le message porte aussi le fait qu'il s'agit d'une erreur dans le contexte OIDC :
```suggestion
  constructor(message = 'Mandatory information returned by the identify provider about the user is missing.', code, meta) {
```

Reply:
Done